README.md

```diff
@@ -58,7 +62,11 @@ Multi-signature transactions are accommodated under-the-hood about 80%, and will
  Windows: [Download](http://www.py2exe.org/)  
 
 * LMDB - database engine, modified to suit Armory's use cases
-[LMDB page](http://symas.com/mdb/)
+ Linux:   Install package `liblmdb-dev` and `python-lmdb`
+ Windows: [LMDB page](http://symas.com/mdb/)
+
+* FastCGI
+ Linux:   Install package `libfcgi-dev`
```

Comment:
This package is distributed as part of the source code as a git submodule. It has custom modifcation.

Reply:
As mentioned elsewhere, it didn't actually get included in the distribution tarball.